PCP/src/main/java/PCP/packets/PCPGroupUsersList.java

```diff
@@ -0,0 +1,141 @@
+/*
+ * this is a school project under "The Unlicence".
+ */
+package PCP.packets;
+
+import PCP.*;
+import java.nio.charset.*;
+import java.util.*;
+
+
+/**
+ *
+ * @author Alessio789
+ * @author gfurri20
+ */
+public class PCPGroupUsersList implements IPCPpacket 
+{
+    private int type;
+    private int listLenght;
+    private String jsonContent;
+
+    public PCPGroupUsersList( int type, int listLenght, String jsonContent ) 
+    {
+        this.type = type;
+        this.listLenght = listLenght;
```

Comment:
list lenght should be calculated and not stored as a variable. 

I suggest using an `ArrayList<String>` and then convert it.
Remember those APIs are meant to be an easy interface to work with.